llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -3196,9 +3196,21 @@ bool AArch64FrameLowering::restoreCalleeSavedRegisters(
     return true;
   }
 
+  SmallVector<RegPairInfo, 8> RegPairsScalable = RegPairs;
```

Comment:
The copy here is not necessary (and `stable_sort`  may do another *pair* of copies,  sigh)